pkg/plugin/util/util.go

```diff
@@ -172,7 +172,17 @@ func ImplementWebhooks(filename string) error {
 	str, err = EnsureExistAndReplace(
 		str,
 		"// TODO(user): fill in your validation logic upon object update.",
-		`if r.Spec.Count < 0 {
+		`if oldRes.Spec.Count != 0 && newRes.Spec.Count < 0 {
```

Comment:
Is this change right?

Reply:
Yes, this is needed to make the compiler happy. `oldRes` is an unused variable otherwise.

Reply:
What this code does is looking for `str,` and replace it with the content.
So, it does not seems right. Where `oldRes` is defined?

Reply:
@jonas-jonas 

Could you please replace it to keep the same previous logic `if res.Spec.Count < 0 {`
So, that if has some issue it will make easier we analyse it

Reply:
Reverted. But please run `make generate` and see for yourself. The code can't be compiled like this.